IDEAS/Buildings/Components/InternalWall.mo

```diff
@@ -2,69 +2,57 @@ within IDEAS.Buildings.Components;
 model InternalWall "interior opaque wall between two zones"
   extends IDEAS.Buildings.Components.Interfaces.PartialOpaqueSurface(
     final use_custom_q50=true,
-    custom_q50=2,
+    custom_q50=if inc==0 or inc == Modelica.Constants.pi then 0 else 2,
```

Comment:
I'm not sure whether I understand the idea here! What's the rationale behind this?